web/src/components/Tooltip.tsx

```diff
@@ -208,18 +208,11 @@ const fixFlow = (element: Element | null, preferredFlow: FlowType, dimensions: V
   }
 
   const checker = spaceCheckers[preferredFlow]
-  if (!checker) {
-    throw new Error('Fallback not found')
-  }
-
   if (checker.check(element.getBoundingClientRect(), dimensions)) {
     return preferredFlow
   }
   const fallback = checker.fallbacks.find((fallbackFlow: FlowType) => {
     const fallbackChecker = spaceCheckers[fallbackFlow]
-    if (!fallbackChecker) {
-      throw new Error('Fallback not found')
-    }
```

Comment:
These checks should stay.

Reply:
It is guaranteed by typescript that there are only valid keys used, so they should not be necessary.

Reply:
spaceCheckers doesnt need to have every key present. The if check can be true if you change the config above.